Mainframe3270/x3270.py

```diff
@@ -209,21 +209,24 @@ def set_screenshot_folder(self, path: str) -> None:
             logger.warn('Screenshots will be saved in "%s"' % self.imgfolder)
 
     @keyword("Take Screenshot")
-    def take_screenshot(self, height: int = 410, width: int = 670) -> str:
+    def take_screenshot(self, height: int = 410, width: int = 670, fnamepfx: str = "screenshot") -> str:
         """Generate a screenshot of the IBM 3270 Mainframe in a html format. The
         default folder is the log folder of RobotFramework, if you want change see the `Set Screenshot Folder`.
 
         The Screenshot is printed in a iframe log, with the values of height=410 and width=670, you
-        can change this values passing them to the keyword.
+        can change these values by passing them to the keyword.
+        
+        The file name prefix can be set; default is "screenshot"
```

Comment:
I would prefer it this phrasing:
The file name prefix can be set, the default is "screenshot".

The semicolon reminds me a bit of programming in this context, the natural syntax sounds more suited towards the user.